mopidy/backend.py

```diff
@@ -31,38 +71,40 @@ class Backend:
     #: The library provider. An instance of
     #: :class:`~mopidy.backend.LibraryProvider`, or :class:`None` if
     #: the backend doesn't provide a library.
-    library = None
+    library: Optional["LibraryProvider"] = None
 
     #: The playback provider. An instance of
     #: :class:`~mopidy.backend.PlaybackProvider`, or :class:`None` if
     #: the backend doesn't provide playback.
-    playback = None
+    playback: Optional["PlaybackProvider"] = None
```

Comment:
I thought the use of `from __future__ import annotations` meant there was no need to quote forward-identifiers like this.

Reply:
You're right. I'll fix.